test/dash/dash_parser_content_protection_unit.js

```diff
@@ -741,3 +741,105 @@ describe('DashParser ContentProtection', function() {
     await Dash.testFails(source, expected);
   });
 });
+
+describe('In-manifest PlayReady and Widevine', function() {
+  let ContentProtection = shaka.dash.ContentProtection;
```

Comment:
Please update the tests to use `const` over `let` whenever possible.

Reply:
Fixed